src/operationsClientBrowser.ts

```diff
@@ -37,10 +37,11 @@ import {createApiCall} from './createApiCall';
 import {PageDescriptor} from './descriptor';
 import {ProjectIdCallback} from 'google-auth-library/build/src/auth/googleauth';
 import {GoogleAuth} from 'google-auth-library';
-
+import {ClientStubOptions} from './grpc';
```

Comment:
No need to have this import here since its usage is commented out

Reply:
ClientStubOptions is also referenced on line 89:
`const opts: OperationsClientOptions & ClientStubOptions`, so I think it still needs to be imported.

Reply:
OK!